mirrulations-work-server/src/mirrserver/work_server.py

```diff
@@ -255,6 +255,7 @@ def put_results(workserver, data):
     job_id = data['job_id']
     workserver.redis.hdel('jobs_in_progress', job_id)
     write_results(results[0], data['directory'], data['results'])
+    print(f"Writing job to {data['directory']}")
```

Comment:
Since you put the output *after* the save line, past tense would actually be better.  Also, I think it would be good to give the filename